qemu/qemu.go

```diff
@@ -863,7 +872,11 @@ func (scsiCon SCSIController) QemuParams(config *Config) []string {
 	var qemuParams []string
 	var devParams []string
 
-	devParams = append(devParams, fmt.Sprintf("virtio-scsi-pci,id=%s", scsiCon.ID))
+	if runtime.GOARCH == "s390x" {
+		devParams = append(devParams, fmt.Sprintf("virtio-scsi-ccw,id=%s", scsiCon.ID))
```

Comment:
you can declare a variable for `virtio-scsi-pci` and change it value to `virtio-scsi-ccw` at at `init()`